etna/core/blocks/image.py

```diff
@@ -40,47 +41,8 @@ def __init__(
         super().__init__(required=required, help_text=help_text, **kwargs)
 
     def get_api_representation(self, value, context=None):
-        if value:
-            jpeg_image = value.get_rendition(
-                f"{self.rendition_size}|format-jpeg|jpegquality-{self.jpeg_quality}"
-            )
-            webp_image = value.get_rendition(
-                f"{self.rendition_size}|format-webp|webpquality-{self.webp_quality}"
-            )
-
-            return {
-                "id": value.id,
-                "title": value.title,
-                "image_jpeg": {
-                    "url": jpeg_image.url,
-                    "full_url": jpeg_image.full_url,
-                    "width": jpeg_image.width,
-                    "height": jpeg_image.height,
-                },
-                "image_webp": {
-                    "url": webp_image.url,
-                    "full_url": webp_image.full_url,
-                    "width": webp_image.width,
-                    "height": webp_image.height,
-                },
-                "transcript": (
-                    {
-                        "heading": value.get_transcription_heading_display(),
-                        "text": value.transcription,
-                    }
-                    if value.transcription
-                    else None
-                ),
-                "translation": (
-                    {
-                        "heading": value.get_translation_heading_display(),
-                        "text": value.translation,
-                    }
-                    if value.translation
-                    else None
-                ),
-            }
-        return None
+        serializer = DetailedImageSerializer()
+        return serializer.to_representation(value)
```

Comment:
I think this is making renditions of all picture elements on the page and cropping them in the process

<img width="933" alt="image" src="https://github.com/nationalarchives/ds-wagtail/assets/2356642/950ada28-0f6d-45bb-842f-faf6956acd00">

Somewhere, these need to be set as a max dimension of 900px (https://nationalarchives.github.io/design-system/styles/images/#picture-elements)

Reply:
The same goes for images in highlight galleries